components/change_url_modal/change_url_modal.tsx

```diff
@@ -146,7 +146,7 @@ export default class ChangeURLModal extends React.PureComponent<Props, State> {
 
     onSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
         e.preventDefault();
-        const url = this.urlInput.current ? this.urlInput.current.value : '';
+        const url = this.urlInput?.current?.value || '';
```

Comment:
Still converting a potential ```null``` value to a string here since the ```cleanUpUrlable``` function only takes strings